### PR TITLE
CPI schema definitions

### PR DESCRIPTION
**Pull request by intrepiditee**
This PR:
1. renames BEAProductEnum to EconomicProductEnum and updates all references to it
2. adds the schema definitions for the CPI import #79

### CODE REVIEW

File: schema/bls.mcf [hunk 1]
```diff
@@ -15329,3 +15329,34 @@ name: "State and local government education"
...
+description: "The reference base is December 1999 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:UrbanConsumer
```
Comment (tjann) [inline schema/bls.mcf:15352]: I would consider either adding a Consumer subclass that UrbanConsumer and UrbanWageEarnerAndClericalWorker can go under; or even an Enum. (Enums can be in popType). WDYT?

Also flagging @pradh 's attention.

Comment (intrepiditee) [reply schema/bls.mcf:15352]: Good idea. I will add a Consumer subclass of Person

File: schema/dcschema_enum_classes.mcf [hunk 2]
```diff
@@ -1,9 +1,9 @@
-Node: dcid:BEAProductEnum
-name: "BEAProductEnum"
+Node: dcid:EconomicProductEnum
+name: "EconomicProductEnum"
 typeOf: schema:Class
 description: "Enumerations of Products and Services used to determine Consumer Price Index (CPI) or price change over time for a market basket of consumer goods and services"
```
Comment (tjann) [inline schema/dcschema_enum_classes.mcf:4]: Super nit: period at the end

File: schema/measurement_methods.mcf [hunk 3]
```diff
@@ -36,3 +36,21 @@ name: "ConstantMaturityRate"
 typeOf: dcs:BondYieldMeasurementEnum
 description: "Indicates that the yield data are extrapolated against a yield curve, which relates the yield on a security to its time to maturity, to estimate the yield of the bond if it had exactly some maturity"
 descriptionUrl: "https://www.investopedia.com/terms/c/cmtindex.asp"
+
+Node: dcid:ChainingEnum
+name: "ChainingEnum"
+typeOf: schema:Class
+subClassOf: schema:MeasurementMethodEnum
+description: "An enumeration of chained and unchained price index measurement methods"
```
Comment (tjann) [inline schema/measurement_methods.mcf:44]: Super nit: period at the end, ditto below

File: schema/dcschema_enum_classes.mcf (hunk 2 above)
Comment (intrepiditee) [reply schema/dcschema_enum_classes.mcf:4]: added

File: schema/measurement_methods.mcf (hunk 3 above)
Comment (intrepiditee) [reply schema/measurement_methods.mcf:44]: added

File: schema/bls.mcf [hunk 4]
```diff
@@ -15329,3 +15329,43 @@ name: "State and local government education"
...
+
+Node: dcid:IndexPointBasePeriod1982-84=100
+name: "IndexPointBasePeriod1982-84=100"
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is 1982 to 1984 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriod1999-12=100
+name: "IndexPointBasePeriod1999-12=100"
+typeOf: dcs:UnitOfMeasure
```
Comment (tjann) [inline schema/bls.mcf:15348]: @intrepiditee I think there's some weird spacing after UnitOfMeasure here. Can you check?

Comment (intrepiditee) [reply schema/bls.mcf]: You are right. I removed the spaces

File: schema/bls.mcf [hunk 5]
```diff
@@ -15329,3 +15329,43 @@ name: "State and local government education"
...
+description: "The reference base is December 1999 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
```
Comment (pradh) [inline schema/bls.mcf:15350]: Should we expand out the symbols I wonder:
- IndexPointBasePeriod1982To84Equals100
- IndexPointBasePeriodDec1982Equals100

File: schema/measurement_methods.mcf [hunk 6]
```diff
@@ -37,6 +37,24 @@ typeOf: dcs:BondYieldMeasurementEnum
 description: "Indicates that the yield data are extrapolated against a yield curve, which relates the yield on a security to its time to maturity, to estimate the yield of the bond if it had exactly some maturity"
 descriptionUrl: "https://www.investopedia.com/terms/c/cmtindex.asp"
 
+Node: dcid:ChainingEnum
```
Comment (pradh) [inline schema/measurement_methods.mcf:40]: ChainingEnum seems to general a type?

Should we make it more specific, like PriceIndexChainingEnum

Comment (intrepiditee) [reply schema/measurement_methods.mcf]: changed to PriceIndexChainingEnum

File: schema/bls.mcf (hunk 5 above)
Comment (intrepiditee) [reply schema/bls.mcf:15350]: I am down but I need a confirmation

Comment (tjann) [reply schema/bls.mcf:15350]: I think that sounds like a good idea.

Comment (intrepiditee) [reply schema/bls.mcf:15350]: changed

**Comment by intrepiditee** [comment]
Can this PR be merged?

**Pull request merged**